pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -78,6 +78,7 @@ const (
 	k8sDashboardKvNamespace              = "dashboard-cleanup"
 	k8sDashboardKvLastResourceVersionKey = "last-resource-version"
 	provisioningConcurrencyLimit         = 10
+	listAllDashboardsLimit               = 100000
```

Comment:
Unsure what are the consequences on unified storage for pulling 100k items max in a row, if this is reduced to ~5k, is it much slower to fetch 100k items?

Reply:
I believe this limit refers to the entire query including all the pages. I.e. if I change this to 5k, we will only get 5k results from our paging.

Reply:
Aha, I thought it was the limit PER page 😄

Reply:
That seems to be completely random and determined under the hood; I seem to get between 2-3k resources with each page.